signer/sign_beacon_block_test.go

```diff
@@ -195,6 +196,63 @@ func TestBenchmarkBlockProposalCapella(t *testing.T) {
68a4e258208203bbab16a82bb897542c66067d34f0040a2f4765945e47c23f32c47e81b286eec6c07a00418010f0b0ed027f641ec3da43968c2930a0bfbba14b4204ada2836e5f7257a691fba717dba5bd76c63742c0280287eb1628add15a57de5b9b9566fe7c3f8f2d85deffd071cbfc4aac3ea17c9986d6970835da7ecf2fa350c3ddf7651c94c531fe165762f8bdf28121796cb48a78efce2e2f1032c61fa0d4ba2beb144eda6d5abecd29ee6b843ad4dcddd2eaf941641f87b650415fb27cfdff6aa17c7cffd5532d9f1cbdafed32015e1d64fd177172549edefca08600ee5d96df9a9c53386f0798f6a410a60d8f69e0534b669d7ab60c9e69355b2379a71dc75ed3525bfd4ee724a9ea4467df494782fccb772e86e88378d3f2968db04d0c0d9a735a64f93e6ea2aeab598e9ecce81a69bdaae94c1733134520980b791dd844d9e0e74d6cb2be9267df5805c0fb2f97611716d820c9223e4112e0e33ee6cbe84b24f4a1c9dae03b52e7a01daa23d384e9679014e0e39d1589f332cd1da38184dd128107271900306df6ba922b89a7eed7b59f2fb92b40eb24ed4c6590550355786a6f7c81b9ea9356f8efe94253f0b9fdf5f1c7cf95f089f4926478fa3979f5d03417873cbcedf5abe883da4f95762e8df227c7e6420cf2ca34b9ea91a92ef75e04abd0f802236aa0a29948544f858b3f9a17f57fa9a56788b81f1f6d512130a095e93c7c5a78d1ae9dcc9f84e7d5a7794cee82b00685714ad4f70f1dc972c86311a21d697988ecd1206b6d840b0a58937636221eb6584889d034d72c460236aab198e9470d53b9d2e475be80eb7c186a681f52bac960030e6db2d16f84a5f2e24513c74707ed68d0dd2d80992fb2873402a7be81d4affb1a46e098e7faef0dda23bad6a105669c7f24a673d9ae47eb22644192f178725380042c63702a99d85aad7f863a5afdb0b0bebd2329718323e7cdc83908d0aaa57c1829826283520d8e6c8643b8d74a8c66fc4dd07628c7554de6fdff9b51c351212c3022181502c6f235ec99e2f03267dd5424765080e96a85d15cb17c326408da582a4dcecc14665e57148146cce2df32e0607a5603b633222a7c9bd5191756c219793f101506202f525f586240b1b437ca08374f7d5fd69e232d0d364f3ef962376b3a8e66d84009b1d4a7064700476d1920ca3549e44f05c2b0cebd1eb12d1ede06dec5f4201308a9758a950963eca7354945000f9db74355999d24f6b00b569f412f9f145221f848279283be1ecbdbaac58f8ca0627d8b87a4442835357763bc5971dd321c87540584a10711b38e894cc45ed55d6741c978e026e6d0e5e665c88d792216f236c0658de098e3c084a458a79b037a0888a58a6a659f6b8b367e07cba21b00109de651505c0cba21790b408bac425a2d9179f6a263d0e28b43fee135dd2196ca6bd848a909230f1e9087b15a069adaaf6962c94c6608c1e17fbfe0a66108e2232cb315263821c88c0f4c4759c3bf62adad5a249d2f80484470bebfdade20274f3fb77135e373190c67aba6d3d2a9bcdd6aa42a345750748d4f9b9779db41b30f52cfd23c5af0a19b40188a30a03b3afb557150a3d293da4222fcfa61456e95187d35605bae235c88eae336963cdac4c38729fc17707991b753e4c62d9d78abcf73aa533554b52e7aa1fbe00229d3f56a0b3e1827326eeaed644d776e0d011d9adbd69adbcd83600458ac23743c6b423105faeb0b658e3214bcd4231b19ae8f2570873839986ded21020a2b8e306a0b8a71d3c88c48a812149383dd1f71a99d6d5a2b19078e1b0a4530f9a59a5ceb73aeb29f7629657ed101b8bbeef8c24567933daec8d57ed379a63711fc0f4aacdf2db513ef0439811fa0d66b2be6166eda85b3d92ec0293a66544021ebb15095e7ff8e17b3133204af2d3c5878b691f077f859981ff5e41d769da367fc60c09bcfaef6286291985f94d65a257881b07ecf26923d296142cb6cde7fa5d53d048f9d8cc073eea785e26c0419ec80977e226647ae4f454f9b5ca8735ffe4988320cea8f0129eb7bede341923aa78e010faed31a6235fdeb32d536687dda933cc70d843f4e19810a4084b2f95c3c
```

Comment:
where is this taken from?

Reply:
there is a comment above the func testing this block